public/js/products.js

```diff
@@ -1,85 +1,70 @@
-(function(item, cartCountElement) {
-    [].forEach.call(item, element => {
-      element.addEventListener('click', event => {
-        event.preventDefault();
+function cartAPI(url, productId) {

```

Comment:
The function name doesn't signify correctly what it does? Shouldn't it be updateCartAPI ??